build_runner/lib/src/build_script_generate/bootstrap.dart

```diff
@@ -187,13 +189,19 @@ final _previousLocationsFile = File(
 /// pre-emptively resolve them by resnapshotting, see
 /// https://github.com/dart-lang/build/issues/1929.
 Future<bool> _checkImportantPackageDeps() async {
-  var currentLocationsContent = _importantPackages
-      .map((pkg) => Isolate.resolvePackageUri(
+  var currentLocationsContent = await Stream.fromIterable(_importantPackages)
```

Comment:
Nit: we don't need a stream here - `Future.wait` would be a bit faster/more efficient.

```
var currentLocationsContent = (await Future.wait(_importantPackages
    map((pkg) => Isolate.resolvePackageUri(
        Uri(scheme: 'package', path: '$pkg/fake.dart'))))
    .join('\n');

Reply:
Good point; I had that initially, but changed it since I wasn't happy with the dartfmt output 😅: 
```dart
  var currentLocationsContent = (await Future.wait(_importantPackages.map(
          (pkg) => Isolate.resolvePackageUri(
              Uri(scheme: 'package', path: '$pkg/fake.dart')))))
      .join('\n');
```
I'll update this and break it up into two statements